CONTRIBUTING.md

```diff
@@ -4,34 +4,53 @@
 
 1. Pop some popcorn :popcorn: (or whatever you enjoy :lollipop: :champagne: :milk_glass: :wine_glass: :tropical_drink: :chocolate_bar: :beer:)
 2. Fork
-3. Write a solution, in the language of your choice. Example: `vim day-03/solutions/super-optimized.py`
-4. Test your solution.
+3. Write a solution, in the language of your choice. Example: `vim days/day-03/solutions/super-optimized.py`
+4. Test your solution
 
 ```sh
-cd days/day03
-../../languages/python.sh input.txt output.txt solutions/super-optimized.py
+$ ./lang/python.sh "days/day-03/solutions/super-optimized.py" "days/day-03/io/my.input days/day-03/io/my.output"
+
+# ..or use the short-hand
+$ ./lang/python.sh "days/day-03/**/*.py" "days/day-03/io/*"
+
+cat INPUT | python3  day03.klyve.py          13ms       ✅
+cat INPUT | python3  day03.preng.py          11ms       ✅
+cat INPUT | python3  one-liner.py            12ms       ✅
 ```
 
-5. Add the test to `day03/test.sh`, to let the CI know how to test your solution.
-6. Make a Pull Request to the `main` branch.
-7. One of the maintainers will merge when the tests pass!
-8. Remember to have fun :tada:
+5. Make sure `days/day03/test.sh` tests your solution.
+
+6. To test the whole day do:
+```sh
+./days/day03/test.sh
+# or
+make test.day03
+# or
+make docker.day03
+```
+
+7. When you are happy with local testing, make a Pull Request to the `main` branch.
+8. One of the maintainers will merge PR's, at the of each day.
```

Comment:
One of the maintainers will merge PR's, at the ................. of each day.

Reply:
:eyes: Well spotted. Since I already merged, I will commit a fix for this directly :+1: